### ISSUE TITLE
What is with the lonely reference to "binary-blobs/" in the README.md file?

### ISSUE THREAD

**Issue by rpjday** [newren/git-filter-repo#1]
Reading the README.md file, and partway down, there is:

"We also want to know how much smaller this extracted repo is without the binary-blobs/ directory in it."

Except there is no further reference to that binary-blobs directory. Is there supposed to be?

**Comment by newren** [comment]
I think I initially wrote up the example removing a single directory ('binary-blobs/') rather than retaining only a single directory ('src/') and missed updating one of the references.  I've got a few other README changes waiting queued up for after I finish the python3 switchover.  Thanks for taking a look and pointing this out, I'll fix it up soon-ish (by early May).

**Comment by rpjday** [comment]
I'm more than happy to check over the end result after the Python 3 switch, as I want to add this utility to one of the Git classes I teach as a replacement for git-filter-branch.

**Comment by newren** [comment]
That'd be great.  I'll give you a heads up, though it may end up getting delayed as I am a bit under the weather.

**Comment by newren** [comment]
Well, that took a lot longer than expected; sorry for the delay.  I finally updated the README with background notes, lots of examples, and information on internals and limitations.  I need to double check that the examples I put in the README actually work (I just wrote them off the top of my head and although I'm pretty sure they work I might have typos or syntax errors), but thought I'd give you a heads up.  Also, this finally fixes this issue.  :-)

Let me know if you have questions, if anything is unclear, if you have improvements suggestions, etc.
